<?xml version="1.0" encoding="UTF-8" ?>

<testsuite failures="1" time="14.482" errors="0" skipped="0" tests="4" name="models.LightningTalkTest">
  <testcase classname="models.LightningTalkTest" name="delete" time="4.952">
      </testcase>
  <testcase classname="models.LightningTalkTest" name="getShowUrl" time="1.74">
      </testcase>
  <testcase classname="models.LightningTalkTest" name="hasVoteFrom" time="2.168">
      </testcase>
  <testcase classname="models.LightningTalkTest" name="findLinkedWith" time="2.685">
      </testcase>
  <testcase classname="models.LightningTalkTest" name="getNumberOfVotes" time="2.937">
                  <failure type="" message="Failure, expected:&lt;2&gt; but was:&lt;0&gt;">
        <![CDATA[
          In /test/models/LightningTalkTest.java, line 36
          
                      assertEquals(2, lt.getNumberOfVotes()); :
          
        ]]>
        </failure>
        }
                  </testcase>
</testsuite>
